framework-plugins/lisk-framework-chain-connector-plugin/src/schemas.ts

```diff
@@ -65,8 +65,8 @@ export const validatorsDataSchema = {
 	},
 };
 
-export const chainConnectorInfoSchema = {
-	$id: '#/plugins/chainConnector/info',
+export const blockHeadersInfoSchema = {
+	$id: '/lisk/plugins/chainConnector/blockHeader',
```

Comment:
Sounds better if called `blockHeaders` or `blockHeadersInfo` because it stores multiple blockHeaders